src/fetch.js

```diff
@@ -57,6 +59,7 @@ function createFetchMiddleware ({ rpcUrl, originHttpHeaderKey }) {
 
 function checkForHttpErrors (fetchRes) {
   // check for errors
+  // eslint-disable-next-line default-case
   switch (fetchRes.status) {
```

Comment:
I could not figure out a default case to define for this case, so I disabled the error for this lint. Is there a default case that can be definedhere?